app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -38,25 +43,29 @@ class HomeMenu(
     private val onHighlightPresent: (BrowserMenuHighlight) -> Unit = {}
 ) {
     sealed class Item {
+        data class Back(val viewHistory: Boolean) : Item()
+        data class Forward(val viewHistory: Boolean) : Item()
+
+        object Bookmarks : Item()
+        object History : Item()
+        object Downloads : Item()
+        object Extensions : Item()
+        object SyncTabs : Item()
+        data class RequestDesktopSite(val isChecked: Boolean) : Item()
         object WhatsNew : Item()
         object Help : Item()
-        object AddonsManager : Item()
         object Settings : Item()
-        object SyncedTabs : Item()
-        object History : Item()
-        object Bookmarks : Item()
-        object Downloads : Item()
-        object Quit : Item()
-        object Sync : Item()
+        object Quit : Item() // waiting
+        object ReconnectSync : Item() // waiting
```

Comment:
I think we should just remove the `// waiting` or file sufficient issues to mention that we should wait these comments. At a glance, it's unclear what we are waiting for since I can see `ReconnectSync` is being used, but maybe it's just `Quit`.

Reply:
Oh shoot, good catch!